site/components/ViewAllContents.tsx

```diff
@@ -698,10 +700,22 @@ const ViewAllContents = ({
             ...filterServices(servicesTndsData),
         ]);
 
+        setCombinedServicesList(combinedServices);
+
         setOperatorsList(operators);
         setServicesList(combinedServices);
     };
 
+    useEffect(() => {
```

Comment:
Then we can streamline this useEffect to be something like:

```ts
setServicesList(
    servicesDataSource === "all"
        ? combinedServicesList
        : combinedServicesList.filter((service) => service.dataSource === servicesDataSource),
);
```

Reply:
done